frontend/src/utils/util.ts

```diff
@@ -36,3 +36,7 @@ export const decryptVault = (passwd: string, vault: Buffer): Buffer => {
 
   return decrypted;
 };
+
+export function generateRandomNumber(): number {
```

Comment:
Shouldn't we use a crypto package for random numbers?
```
window.crypto.getRandomValues(array);
```
It is safer than math random 

https://developer.mozilla.org/en-US/docs/Web/API/Crypto/getRandomValues